### PR TITLE
Documentation on generating cordova using maven command

### PR DESCRIPTION
**Pull request by sujith-simon**

### CODE REVIEW

**Review by swethakundaram — APPROVED** [review]

File: learn/hybrid-mobile/export-cordova-zip.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+A Cordova zip will be used to create a mobile application. WaveMaker allows you to export a cordova zip from a *mobile project* so that
+you can use it to generate a *mobile application*.
+
+Following are the two approaches which can be followed to create a cordova zip 
+
+## 1. Using WaveMaker studio
+
+1. In the top right menu bar of WaveMaker Studio, export menu is present.  Click on 'Download Cordova zip' menu item.
```
Comment (jvenugopal) [inline learn/hybrid-mobile/export-cordova-zip.md]: I think "Download Cordova zip" should be renamed to "Project As Cordova Zip" similar to how the other ones are named.

File: learn/hybrid-mobile/export-cordova-zip.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+[![](/learn/assets/Cordova_Zip.png)](/learn/assets/Cordova_Zip.png)
+
+2. A dialog with all the cordova configuration will open.
+[![](/learn/assets/Build_Cordova_Zip.png)](/learn/assets/Build_Cordova_Zip.png) 
+
+3. Provide all necessary configuration and click on 'build' button. A job gets posted to prepare a cordova zip.
```
Comment (jvenugopal) [inline learn/hybrid-mobile/export-cordova-zip.md]: We can talk about few important configuration options to review (like the server url or so).
Also point to another document where the configuration options are talked about.

File: learn/hybrid-mobile/export-cordova-zip.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+4. When the job gets completed, a link will be provided (in the job info) to download the cordova zip.
+
+
+## 2. Using maven command
+
+:::tip
+This an alternative of the above approach. This is useful if you are working outside WaveMaker studio.
+:::
+
+1. Export and extract the WaveMaker project zip if you don't have it already.
```
Comment (jvenugopal) [inline learn/hybrid-mobile/export-cordova-zip.md:37]: you need to talk about the git clone option as well here not just the extracting the project zip.

File: learn/hybrid-mobile/export-cordova-zip.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+2. Execute the command `mvn clean package -Dmobile.serverUrl="<url>"`. Here, **&lt;url&gt;** is the application server url.
+The mobile application will generated from this cordova will make backend calls to the url specified. (*Refer the below image*)
+
+![maven command to generate cordova](/learn/assets/cordova_maven_command.png)
+
+3. After executing the above command, you can find the cordova zip in the target folder. (*Refer the below image*)
```
Comment (jvenugopal) [inline learn/hybrid-mobile/export-cordova-zip.md]: could you talk about the zip is named & also highlight the file in the image.

**Review by jvenugopal — CHANGES_REQUESTED** [review]

File: learn/hybrid-mobile/export-cordova-zip.md (hunk 1 above)
Comment (sujith-simon) [reply learn/hybrid-mobile/export-cordova-zip.md]: @PatnalaPriyanka is working on the UI change. Will update the document in next minor release

**Pull request merged**